### PR TITLE
 355-10-prolongation-wrong-error-message

### LINKED ISSUE #355: [10] Prolongation wrong error message
After login, switched to load service and clicking  prolonge button, "PLN" is missing in the message.
image
 Once Return button is clicked, additional {0} appears in the message and the rest is ok.
image
This bug is the result of [20] Prolongation #317 implementation.

### PR DESCRIPTION
**Pull request by JPerzanowski**

### CODE REVIEW

File: elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java [hunk 1]
```diff
@@ -472,7 +472,7 @@ public static BlockReason getByNumber(String blockReason) {
 	public static final Boolean DATA_BASE_REGISTER_DEFAULT_FLAG = false;
 	public static final Integer DATA_BASE_ADMIN_ID = 0;
 	public static final String DATA_BASE_ADMIN_LOGIN = "admin";
-	public static final String DATA_BASE_ADMIN_PASSWORD = "f7e3c24e1a04758097f69be41aa3cf18";
```
Comment (JaRutkowski) [inline elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java]: Please remove it as it's a local change. 🗑️

File: elibrary-hibernate/pom.xml [hunk 2]
```diff
@@ -40,12 +40,12 @@
 					<propertyFile>liquibase.properties</propertyFile>
 					<changeLogFile>src/main/resources/db-changelog-master.xml</changeLogFile>
 					<!-- comment if running locally -->
-					<url>
```
Comment (JaRutkowski) [inline elibrary-hibernate/pom.xml]: Please remove it as it's a local change. 🗑️

File: elibrary-hibernate/src/main/resources/db-changelog-master.xml [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
 	<!-- Pre-processing -->
 	<!-- Increment id of changeSet -->
-	<include file="src/main/resources/db/changelog/truncate-all-cascade-changelog.xml" relativeToChangelogFile="false"/>
```
Comment (JaRutkowski) [inline elibrary-hibernate/src/main/resources/db-changelog-master.xml]: Please remove it as it's a local change. 🗑️

File: elibrary-hibernate/src/main/resources/db-changelog-master.xml [hunk 4]
```diff
@@ -54,5 +54,6 @@
 	<!-- it is possible to run DML on local: -->
 	<!--   - liquibase plugin part - db configuration - should be commented in pom.xml; -->
 	<!--   - all data should be truncated; -->
-	<!--   - last version changelog should be applied using mvn liquibase:update using dev profile. -->
```
Comment (JaRutkowski) [inline elibrary-hibernate/src/main/resources/db-changelog-master.xml]: Please remove it as it's a local change. 🗑️

File: elibrary-hibernate/src/main/resources/db/changelog/db-changelog-20210816101031UTC.xml [hunk 5]
```diff
@@ -1,5 +1,14 @@
 <?xml version="1.1" encoding="UTF-8" standalone="no"?>
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="Mój (generated)" id="1591996331364-0">
```
Comment (JaRutkowski) [inline elibrary-hibernate/src/main/resources/db/changelog/db-changelog-20210816101031UTC.xml]: Please remove it as it's a local change. 🗑️

File: pom.xml [hunk 6]
```diff
@@ -31,7 +31,7 @@
 		<org.bidib.org.oxbow.version>1.2.2</org.bidib.org.oxbow.version>
 		<net.coderazzi.version>5.5.1</net.coderazzi.version>
 		<com.sun.mail.version>1.6.2</com.sun.mail.version>
-		<org.projectlombok.version>1.18.12</org.projectlombok.version>
```
Comment (JaRutkowski) [inline pom.xml]: Please remove it as it's a local change. 🗑️

**Review by JaRutkowski — CHANGES_REQUESTED** [review]
PSB

File: elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java (hunk 1 above)
Comment (JPerzanowski) [reply elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java]: I did it.

File: elibrary-hibernate/pom.xml (hunk 2 above)
Comment (JPerzanowski) [reply elibrary-hibernate/pom.xml]: I did it.

File: elibrary-hibernate/src/main/resources/db-changelog-master.xml (hunk 3 above)
Comment (JPerzanowski) [reply elibrary-hibernate/src/main/resources/db-changelog-master.xml]: I did it.

File: elibrary-hibernate/src/main/resources/db-changelog-master.xml (hunk 4 above)
Comment (JPerzanowski) [reply elibrary-hibernate/src/main/resources/db-changelog-master.xml]: I did it.

File: elibrary-hibernate/src/main/resources/db/changelog/db-changelog-20210816101031UTC.xml (hunk 5 above)
Comment (JPerzanowski) [reply elibrary-hibernate/src/main/resources/db/changelog/db-changelog-20210816101031UTC.xml]: I did it.

File: pom.xml (hunk 6 above)
Comment (JPerzanowski) [reply pom.xml]: I did it.

**Review by JaRutkowski — CHANGES_REQUESTED** [review]
Remove, meaning revert. If u've removed the code from the sources, then the code will not compile.

**Review by JaRutkowski — APPROVED** [review]

**Pull request merged**